vault/token_store.go

```diff
@@ -102,14 +107,11 @@ func NewTokenStore(c *Core, config *logical.BackendConfig) (*TokenStore, error)
 	t := &TokenStore{
 		view:               view,
 		cubbyholeDestroyer: destroyCubbyhole,
+		logger:             c.logger,
+		policyLookupFunc:   c.policyStore.GetPolicy,
```

Comment:
By doing this here you are skipping the check for nil; IIRC this check is there on purpose for some test reasons.